Tests/E2E/IntegrationTestFixtureCI.cs

```diff
@@ -568,6 +573,16 @@ public override void SetupTestDevices()
                 AppKey = "00000000000000000000000000000031",
                 IsIoTHubDevice = true
             };
+
+            Device32_ABP = new TestDeviceInfo()
+            {
+                DeviceID = "0000000000000032",
+                AppSKey = "00000000000000000000000000000032",
```

Comment:
done in 1ffe36f762585f0fded9a5e2c028ebdfc09e58ca

Reply:
I reverted this change because when using the helper methods the test would fail sometimes (both locally and in the CI) due to the device not being recognized and all message being dropped. I will open a bug for this as it shouldn't be the case, right now I'm not sure what the reason is.